backend/pom.xml

```diff
@@ -14,6 +14,7 @@
   <name>Debezium UI Backend</name>
 
   <properties>
+    <quarkus.package.type>uber-jar</quarkus.package.type>
```

Comment:
Oh, what's the reason for the uber-JAR? This was consciously using the multi-layer approach before.

Reply:
See https://issues.redhat.com/browse/DBZ-4621